app/Resources/views/Version/new.html.twig

```diff
@@ -0,0 +1,18 @@
+{% extends '::layout.html.twig' %}
+
+{% block page_title -%} New Version {% endblock %}
+
+{% block content %}
+    
+    {{ form(form) }}
+
+
+
```

Comment:
@isaias1503 if the code is needed please un-comment the code, other wise delete it